code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -59,6 +59,20 @@
 	var/obj/machinery/door/D = target
 	D.locked = FALSE
 	update_icon()
+	to_chat(user, "<span class='notice'>[src] flickers and disappears right before your eyes!</span>")
+	playsound(loc, 'sound/magic/Staff_Change.ogg', 50, TRUE, -1)
+	. = ..()
+	qdel(src)
+
+/obj/item/card/emag/skeleton_key
+	name = "skeleton key"
+	desc = "A strange bone key with a weak electromagnetic aura."
+	icon_state = "skeleton_key"
+
+/obj/item/card/emag/skeleton_key/afterattack__legacy__attackchain(atom/target, mob/user, proximity)
```

Comment:
we dont allow for new implementations of legacy attack chain procs. I think someone else has a migration PR open for it, so just wait a bit, but this will eventually need to be moved over.